configs/satellite_config.yaml

```diff
@@ -17,23 +17,30 @@
 
 logger:
   log_pattern: "%time [%level][%field] - %msg"
-  time_pattern: "2006-01-02 15:04:05.001"
+  time_pattern: "2006-01-02 15:04:05.000"
   level: "info"
 
+telemetry:
+  namespace: namespaces
+  service: service1
+  instance: instance1
+
 sharing:
+  common_config:
+    pipe_name: sharing
   clients:
     - plugin_name: "grpc-client"
       k: v
   servers:
     - plugin_name: "grpc-server"
       k: v
-namespaces:
+pipes:
```

Comment:
What does `pipes` mean?

Reply:
the namespace concept is similar to the Kubernetes namespaces, so I change the original namespaces concept to the new name call pipe. Each pipe has one gatherer, one processor, and one sender.

Reply:
Do you update the doc? I think the `namespace` has been mentioned there.

Reply:
sorry, thx for your notice